<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicio</class>
 <widget class="QMainWindow" name="inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Lab_clinico">
   <widget class="QGroupBox" name="ing_pacientes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>431</height>
     </rect>
    </property>
    <widget class="QLabel" name="Direccion">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Direccion</string>
     </property>
    </widget>
    <widget class="QLabel" name="RUT">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RUT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ingreso_id_paciente">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Edad">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Edad</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ingreso_rut">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ingreso_direccion">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ingreso_correo">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ID_Paciente">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>id_paciente</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ingreso_edad">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>130</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Fecha">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha ingreso</string>
     </property>
    </widget>
    <widget class="QLabel" name="Apellidos">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </widget>
    <widget class="QLabel" name="Ingresar_pacientes">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>INGRESAR PACIENTES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ingresar_paciente">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>280</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar paciente</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/insertar.png</normaloff>../../../../Downloads/insertar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ingreso_apellidos">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="Ingreso_calendario">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>321</width>
       <height>183</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Nombres">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombres</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ingreso_nombres">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Correo">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Correo</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="busc_pacientes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>1091</width>
      <height>431</height>
     </rect>
    </property>
    <widget class="QLabel" name="Buscar">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>BUSCAR PACIENTES</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Buscar_paciente">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>242</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTableWidget" name="Tabla">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>1091</width>
       <height>331</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>id_paciente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombres</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellidos </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>RUT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Direcci√≥n</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Edad</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha de ingreso</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Correo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo de examen</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Resultado</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="ID_paciente">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> id_paciente</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Buscar_boton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>126</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>BUSCAR</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/search_icon_125165.png</normaloff>../../../../Downloads/search_icon_125165.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ing_examenes">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>531</width>
      <height>431</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="ingresar_examen">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>360</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingresar examen</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/insertar.png</normaloff>../../../../Downloads/insertar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="Resultado">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Resultado</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Box_examenes">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>146</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>examen_1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>examen_2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>examen_3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>examen_4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>examen_5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="pacientes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>85</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>id_paciente</string>
     </property>
    </widget>
    <widget class="QLabel" name="Ing_examen">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>INGRESO EXAMEN</string>
     </property>
    </widget>
    <widget class="QLabel" name="Examen">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de examen</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Ingreso_id_paciente_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="Ingreso_resultado">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>421</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
